omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/EncounterController1_9Test.java

```diff
@@ -500,6 +501,29 @@ public void shouldUnVoidAnEncounter() throws Exception {
 		
 	}
 	
+	@Test
+	public void shouldFetchAllPatientEncounterIncludingVoidedPatientEncountersIfIncludeAllParamIsSetToTrue()
+	        throws Exception {
+		Patient patient = Context.getPatientService().getPatientByUuid(RestTestConstants1_9.PATIENT_WITH_ENCOUNTER_UUID);
+		
+		Encounter enc = Context.getEncounterService().getEncounterByUuid(RestTestConstants1_9.ENCOUNTER_UUID);
+		Context.getEncounterService().voidEncounter(enc, "some reason");
+		Context.getEncounterService().saveEncounter(enc);
+		
+		//list of non voided encounters
+		List<Encounter> encs = Context.getEncounterService().getEncountersByPatient(patient.getGivenName());
```

Comment:
Why are you using the patient name to fetch encounters?

Reply:
corrected it